index.html

```diff
@@ -232,7 +238,7 @@
 						---
 
 						### Attributes
-						Attributes give more information about your element. Some are required for the element to work, such as the `src="img.jpg"` on the `<img>` tag. Some are optional.
+						Attributes give more information about your element. Some are required for the element to work, such as the `src` and `alt` on the `<img>` tag. Some are optional.
```

Comment:
Since we point out later that the alt is not optional, the way this was worded was a bit misleading.